PhysicsTools/TensorFlow/test/testLoading.cc

```diff
@@ -0,0 +1,121 @@
+/*
+ * Graph and session loading tests.
+ * Based on TensorFlow C++ API 1.3.
+ * For more info, see https://gitlab.cern.ch/mrieger/CMSSW-DNN.
+ *
+ * Author: Marcel Rieger
+ */
+
+#include <boost/filesystem.hpp>
+#include <cppunit/extensions/HelperMacros.h>
+#include <stdexcept>
+
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+
+class testLoading : public CppUnit::TestFixture
+{
+    CPPUNIT_TEST_SUITE(testLoading);
+    CPPUNIT_TEST(checkAll);
+    CPPUNIT_TEST_SUITE_END();
+
+public:
+    std::string dataPath;
+
+    void setUp();
+    void tearDown();
+    void checkAll();
+};
+
+CPPUNIT_TEST_SUITE_REGISTRATION(testLoading);
+
+void testLoading::setUp()
+{
+    dataPath = std::string(getenv("CMSSW_BASE")) + "/test/" + std::string(getenv("SCRAM_ARCH"))
+        + "/" + boost::filesystem::unique_path().string();
+
+    // create the graph
+    std::string testPath = std::string(getenv("CMSSW_BASE")) + "/src/PhysicsTools/TensorFlow/test";
```

Comment:
if I'm not mistaken, these will fail in release environment.
Is it supposed to be ```SCRAMRT_LOCALRT```
@smuzaffar please check/remind

Reply:
Any updates on this matter? `SCRAMRT_LOCALRT` is not set in a local env.

Reply:
I guess it's just a unit test that may start failing, which is not really a blocker.
So, we could fix it once it's in the release.

Reply:
@slava77 and @riga this will fail during IB tests and project dev area (where this package is not checkout out). You need to use something like this  https://github.com/cms-sw/cmssw/blob/bab6aa259dcd4c5be70c84fd227203943f7befe6/FWCore/Catalog/test/FileLocator_t.cpp#L52

i.e. if exists CMSSW_BASE+yourfile then use it otherwise use CMSSW_RELEASE_BASE+yourfile
@Dr15Jones , do you know if we already have any utility function in FW for this?

Reply:
Ok, will do.